doc.config.json

```diff
@@ -0,0 +1,18 @@
+{
+  "glob": "modules/**/*.{ts,tsx}",
+  "ignore": [
+    "**/examples/**",
+    "**/spec/**",
+    "**/stories/**",
+    "**/codemod/**",
+    "**/docs/**",
+    "**/css/**",
+    "*.d.ts",
+    "*.js"
+  ],
+  "plugins": [
+    "modules/docs/docgen/plugins/enhancedComponentParser.ts",
+    "modules/docs/docgen/plugins/componentParser.ts",
+    "modules/docs/docgen/plugins/modelParser.ts"
+  ]
+}
```

Comment:
Config file for the new doc parser. It both configures the parser as well as "anchor" file pathing.